InvenTree/order/templates/order/order_wizard/select_parts.html

```diff
@@ -49,7 +51,13 @@ <h4>
                         <select class='select' id='id_supplier_part_{{ part.id }}' name="part-supplier-{{ part.id }}">
                             <option value=''>---------</option>
                             {% for supplier in part.supplier_parts.all %}
-                            <option value="{{ supplier.id }}"{% if part.order_supplier == supplier.id %} selected="selected"{% endif %}>{{ supplier }}</option>
+                            <option value="{{ supplier.id }}"{% if part.order_supplier == supplier.id %} selected="selected"{% endif %}>
+                                {% if show_price %}
+                                    {% call_method supplier 'get_price' part.order_quantity as price %}
+                                    {% include "price.html" with price=price %} - 
```

Comment:
![image](https://user-images.githubusercontent.com/10080325/122540036-93855900-d06b-11eb-9e8c-27d819fc4c0f.png)

This line throws an error immediately for me, I think you need to pass currency to price.html ?

Reply:
@SchrodingersGat that is done in line 7, I think the merge fixed if, could you try again?